ols/customize/rhdh/prompts.py

```diff
@@ -0,0 +1,205 @@
+# There is no need for enforcing line length in this file,
+# as these are mostly special purpose constants.
+# ruff: noqa: E501
+"""Prompt templates/constants."""
+
+from ols.constants import SUBJECT_ALLOWED, SUBJECT_REJECTED
+
+# Default responses
+INVALID_QUERY_RESP = (
+    "Hi, I'm the RHDH Lightspeed assistant, I can help you with questions about RHDH or Backstage. "
```

Comment:
I'm thinking to avoid using the acronym here. Not many outside our circle refer to developer hub as RHDH